### PR TITLE
test(components/flyout): fix flyout visual test inconsistencies

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1453/storybook/)

Component Storybooks:

- [flyout-storybook](https://blackbaud.github.io/skyux-pr-preview/1453/storybooks/flyout-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1453/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1453/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1453/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1453/playground/)

File: apps/e2e/flyout-storybook/src/app/flyout/flyout-responsive.component.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ export class FlyoutResponsiveComponent {
 
   constructor(private mediaQueryService: SkyMediaQueryService) {
     this.mediaQueryService.subscribe((breakpoint) => {
+      console.log(breakpoint);
```
Comment (johnhwhite) [inline apps/e2e/flyout-storybook/src/app/flyout/flyout-responsive.component.ts]: Was this intentional?

Comment (Blackbaud-TrevorBurch) [reply apps/e2e/flyout-storybook/src/app/flyout/flyout-responsive.component.ts]: Nope - removed

**Comment by johnhwhite** [comment]
image
That doesn't look right, right?

**Comment by Blackbaud-TrevorBurch** [comment]
> That doesn't look right, right?

Pushed a change for this.

**Review by johnhwhite — APPROVED** [review]

**Pull request merged**